gradle.properties

```diff
@@ -11,4 +11,6 @@ org.gradle.parallel=true
 org.gradle.configureondemand=true
 kapt.incremental.apt=true
 org.gradle.daemon=true
-org.gradle.configuration-cache=true
\ No newline at end of file
+org.gradle.configuration-cache=true
+
+nextcloudAndroidCommonLib=ab22edd0cf
```

Comment:
nice work and catch @alperozturk96 
Any way we could also put this into the main `build.gradle` file like the other versions?

Reply:
I will move it.